### DISCUSSION TITLE
Adding custom headers to ajax requests

### DISCUSSION THREAD

**Discussion by alhoqbani** [livewire/livewire#1320] [4. General]
Is it possible to add custom headers to ajax requests created by livewire?

I'm using `stancl/tenancy` and would like to send tenant id with every request to identify the tenant. 

Looking at the [source code](https://github.com/livewire/livewire/blob/cd3f7d81288b390ddecabc31e5f180e7ae3fb55e/js/connection/drivers/http.js#L19-L25), it seems to me there is no way to add any headers, is there any workaround to intercept the request and add custom headers?

**Comment by naabster** [comment]
I had the same problem, created a pull request: https://github.com/livewire/livewire/pull/4141
